### ISSUE TITLE
Exception when running on ansible 2.1.0.0

### ISSUE THREAD

**Issue by erikcw** [f500/ansible-dumpall#6]
It worked pre 2.0.

```
TASK [f500.dumpall : Dump all vars] ********************************************
fatal: [localhost]: FAILED! => {"changed": false, "failed": true, "msg": "AnsibleError: Unexpected templating type error occurred on ({{ ansible_managed }}\n\n\nModule Variables (\"vars\"):\n--------------------------------\n{{ vars | to_nice_json }}\n\nEnvironment Variables (\"environment\"):\n--------------------------------\n{{ environment | to_nice_json }}\n\nGROUP NAMES Variables (\"group_names\"):\n--------------------------------\n{{ group_names | to_nice_json }}\n\nGROUPS Variables (\"groups\"):\n--------------------------------\n{{ groups | to_nice_json }}\n\nHOST Variables (\"hostvars\"):\n--------------------------------\n{{ hostvars[inventory_hostname] | to_nice_json }}\n): exceptions must be old-style classes or derived from BaseException, not type"}
```

```yaml
---
- hosts: localhost
  connection: local
  gather_facts: False
  roles:
    - { role: f500.dumpall, dumpall_host_destination: /tmp/ansible_vars_dump_prov/ }
```

**Comment by JoeyG1973** [comment]
This can be fixed for 2.x by changing 
`{{ vars | to_nice_json }}`

to

`{{ hostvars[inventory_hostname] | to_nice_json }}`

Found the answer here:

https://github.com/ansible/ansible/issues/13838

**Comment by ramondelafuente** [comment]
This is fixed in v2.0.2
